### PR TITLE
Update script with new groups

### PR DESCRIPTION
**Pull request by abeljoseph**

### CODE REVIEW

File: google_apis/google_services_manager.py [hunk 1]
```diff
@@ -50,13 +50,13 @@ def download_file_from_drive(self, drive_file_id, return_df=True, remove=False):
...
             file = self.drive_client.CreateFile({'id': drive_file_id})
-            file_name = '{}.csv'.format(file['title'])
+            file_name = file['title']
             file.GetContentFile(file_name, mimetype='text/csv')
             if remove:
                 file.Delete()
             # If set to true, read file as df and return df and delete file
             if return_df:
-                df = pd.read_csv(file_name, na_filter=False)
+                df = pd.read_csv(file_name, na_filter=False, encoding='ISO-8859-1')
```
Comment (simonarocco) [inline google_apis/google_services_manager.py:59]: why did u have to change encoding

File: google_apis/google_services_manager.py [hunk 2]
```diff
@@ -50,13 +50,13 @@ def download_file_from_drive(self, drive_file_id, return_df=True, remove=False):
         # Download file from google drive with specified file id
         try:
             file = self.drive_client.CreateFile({'id': drive_file_id})
-            file_name = '{}.csv'.format(file['title'])
+            file_name = file['title']
```
Comment (simonarocco) [inline google_apis/google_services_manager.py:53]: this doesnt break anything right

**Review by simonarocco — COMMENTED** [review]
a few questions

File: google_apis/google_services_manager.py (hunk 1 above)
Comment (abeljoseph) [reply google_apis/google_services_manager.py:59]: The new files that Nathan uploaded have this encoding — by default it’s UTF-8 but it was throwing an error

File: google_apis/google_services_manager.py (hunk 2 above)
Comment (abeljoseph) [reply google_apis/google_services_manager.py:53]: Nope. The new files are named “GroupA.csv”, etc. so we don’t have to add the extension unlike before. The files are deleted afterwards anyways

**Pull request merged**